build_options.cmake

```diff
@@ -9,10 +9,12 @@
 # http://www.cmake.org/Wiki/CMake_Cross_Compiling
 # https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html
 # http://stackoverflow.com/questions/12802377/in-cmake-how-can-i-find-the-directory-of-an-included-file
-if ( WIN32 )
+if ( WIN32 ) # TODO: add detection of Windows Phone / Windows Universal platform
```

Comment:
that would probably go with an explicit toolchain file (like android and ios)...